src/opm/input/eclipse/Schedule/Action/Condition.cpp

```diff
@@ -247,6 +250,8 @@ int Condition::comparator_as_int() const {
         return 4;
     case Comparator::EQUAL:
         return 5;
+    case Comparator::NOT_EQUAL:
+        return 6;
```

Comment:
How did you select this value?  For what it's worth, the values [go into the restart file](https://github.com/OPM/opm-common/blob/c28f51a11c89a89abea84c9e33310fe6d026f7f0/src/opm/output/eclipse/AggregateActionxData.cpp#L290) and this value may or may not break compatibility.

Reply:
It is not really random. The previously supported operators translate to values 1-5. Naive as I am, I made the new `!=` become 6 in the hope that this would be backwards compatible.

Reply:
`comparator_from_int` is changed consistently. I guess that is used for reading.

Reply:
> I made the new `!=` become 6 in the hope that this would be backwards compatible.

Okay, cool.  It just so happens that `6` is the appropriate value for the "not equals" operator so this PR is good to go from that point of view.

Reply:
Perfect. Thanks for the review.